xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -8582,25 +8582,24 @@ int
 glusterd_get_dummy_client_filepath(char *filepath, glusterd_volinfo_t *volinfo,
                                    gf_transport_type type)
 {
-    int ret = 0;
+    char *suffix;
+    int32_t ret;
 
-    switch (type) {
-        case GF_TRANSPORT_TCP:
-        case GF_TRANSPORT_BOTH_TCP_RDMA:
-            snprintf(filepath, PATH_MAX, "/tmp/%s.tcp-fuse.vol",
-                     volinfo->volname);
-            break;
+    if ((type == GF_TRANSPORT_TCP) || (type == GF_TRANSPORT_BOTH_TCP_RDMA)) {
+        suffix = "tcp-fuse";
+    } else if (type == GF_TRANSPORT_RDMA) {
```

Comment:
It's no longer supported, but it still seems to be allowed (at least in glusterd). I remember some patches to remove RDMA related code, but they were not complete removals.

Since this is a patch only intended for regression test failures, I would prefer to not change any functionality here and defer the removal until it's globally removed from glusterd in another patch.